src/components/Calculator.js

```diff
@@ -1,64 +1,66 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Calculate from '../logic/calculate';
 import './Calculator.css';
 
-export default class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      next: null,
-      operation: null,
-    };
-    this.handleClick = this.handleClick.bind(this);
-  }
-
-  handleClick = (e) => {
+const Calculator = () => {
+  const [state, setState] = useState({
+    total: '0',
+    next: null,
+    operation: null,
+  });
+  const handleClick = (e) => {
     if (!e.target.name) return;
-    const { total, next, operation } = Calculate(this.state, e.target.name);
-    if (!next && !total) this.setState({ total: '0', next, operation });
-    else this.setState({ total, next, operation });
-  }
+    const { total, next, operation } = Calculate(state, e.target.name);
+    if (!next && !total) setState({ total: '0', next, operation });
+    else setState({ total, next, operation });
+  };
+  const btns = [
+    'AC',
+    '+/-',
+    '%',
+    '÷',
+    '7',
+    '8',
+    '9',
+    'x',
+    '4',
+    '5',
+    '6',
+    '-',
+    '1',
+    '2',
+    '3',
+    '+',
+    '0',
+    '.',
+    '=',
+  ];
+  const operations = '+-x÷=';
```

Comment:
- [ ] After entering the first number and pressing an operator(`+-*/`), in the process of entering the second number,  it doesn't display (only the first number is still being displayed), after pressing the equality sign (`=`), it displays the correct results. Please kindly fix the issue as it will be nice to be seeing all the values as you are entering them.

Reply:
Thanks for the suggestions. I have implemented said changes.